Gemfile

```diff
@@ -83,6 +83,7 @@ gem "rails-i18n"
 # Provides helper methods to easily add 'active' tag on links
 gem "active_link_to"
 gem "nice_partials"
+gem "local_time"
```

Comment:
I didn't actually need this library but I'am thinking we may need this anyways eventually!